articles/portal/ptl-ref-release-notes.md

```diff
@@ -377,21 +399,21 @@ Date: 17th May 2017
 
 Feature updates
 
-Details: The 'List All Articles' button in the Knowledge Centre has now been moved to the top of the categories menu for better visibility. 
+Details: The 'List All Articles' button in the Knowledge Centre has now been moved to the top of the categories menu for better visibility.
 
 ## Release 1.55.0
 
 Date: 2nd May 2017
 
 Feature updates
 
-Details: Internet Bandwidth consumption showback available to all customers. Please see the Bandwidth Monitoring FAQ document in the Knowledge Centre for more details. 
+Details: Internet Bandwidth consumption showback available to all customers. Please see the Bandwidth Monitoring FAQ document in the Knowledge Centre for more details.
```

Comment:
link to doc?

Reply:
That doc is three years old, I'm not convinced I'll be able to find it

Reply:
I think [that's](https://docs.ukcloud.com/articles/connectivity/conn-faq-bandwidth-monitoring.html?q=Bandwidth%20Monitoring%20FAQ) the closest thing we now have

Reply:
This is such an old item in the release notes, I don't think we need to bother providing a link